session4/src/itschool/homework/ex1/AreaRectangle.java

```diff
@@ -0,0 +1,18 @@
+package itschool.homework.ex1;
+
+import java.util.Scanner;
+
+public class AreaRectangle {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        float len, bre, area;
```

Comment:
variables declaration on the same line is discouraged and should be avoided because is harder to read and to maintain. So, each of them should be declared on different lines.However, no need for them here (extra code that can be avoided). You could directly declare them at the line where you initialized them.
E.g.: variable `len` can be both declared and initialized directly at line 11:
`float len = scan.nextFloat();`
Apply this change for all 3 vars.